### PR TITLE
added t size rmsea and cfi

### PR DESCRIPTION
**Pull request by LSLindeloo**
fixes https://github.com/jasp-stats/jasp-issues/issues/1837

I am not sure whether to add the t size equivalents of the frequently used cutoff values for the rmsea and cfi, since there is no consensus on these values and the use is controversial

### CODE REVIEW

**Comment by LSLindeloo** [comment]
fixes https://github.com/jasp-stats/jasp-issues/issues/1837

File: R/sem.R [hunk 1]
```diff
@@ -1071,11 +1117,11 @@ checkLavaanModel <- function(model, availableVars) {
     gettext("Expected cross validation index (ECVI)")
   )
   if (length(options[["models"]]) == 1) {
-    fitot[["value"]] <- fmli[[1]][c("rmsea", "rmsea.ci.lower", "rmsea.ci.upper", "rmsea.pvalue",
+    fitot[["value"]] <- fmli[[1]][c("rmsea", "rmsea.ci.lower", "rmsea.ci.upper", "rmsea.pvalue", "rmsea.t",
```
Comment (Kucharssim) [inline R/sem.R:1120]: can you add citations of the original T-size articles to the tables that report these measures?

**Comment by Kucharssim** [comment]
any development @LSLindeloo ? :)

File: R/sem.R [hunk 2]
```diff
@@ -1054,12 +1107,23 @@ checkLavaanModel <- function(model, availableVars) {
     gettext("Relative Noncentrality Index (RNI)")
   )
   if (length(options[["models"]]) == 1) {
-    fitin[["value"]] <- fmli[[1]][c("cfi", "tli", "nnfi", "nfi", "pnfi", "rfi", "ifi", "rni")]
+    fitin[["value"]] <- fmli[[1]][c("cfi", "cfi.t", "tli", "nnfi", "nfi", "pnfi", "rfi", "ifi", "rni")]
+    cfi_t_footnote <- paste0(gettextf("The T-size equivalents of the conventional CFI cut-off values (poor < 0.90 < fair < 0.95 < close) are <b>poor < %s < fair < %s < close</b> for model: ",
```
Comment (Kucharssim) [inline R/sem.R:1111]: this footnote should be all formatted with a `gettextf` + use positional arguments (see https://github.com/jasp-stats/INTERNAL-jasp/issues/2125)

**Review by Kucharssim — CHANGES_REQUESTED** [review]
Just one little thing

**Review by Kucharssim — APPROVED** [review]

**Pull request merged**